lib/webauthn.rb

```diff
@@ -12,17 +12,15 @@
 
 module WebAuthn
   CRED_PARAM_ES256 = { type: "public-key", alg: COSE::Algorithm.by_name("ES256").id }.freeze
-  RP_NAME = "web-server"
-  USER_ID = "1"
-  USER_NAME = "web-user"
   TYPES = { create: "webauthn.create", get: "webauthn.get" }.freeze
 
-  def self.credential_creation_options
+  # TODO: make keyword arguments mandatory in next major version
+  def self.credential_creation_options(rp_name: "web-server", user_name: "web-user", display_name: "web-user", id: "1")
```

Comment:
I think I'll change `id` to `user_id` to avoid confusion with RP ID.

Reply:
Changed in https://github.com/cedarcode/webauthn-ruby/commit/18bb1d299b1354e6bd4e5ca733947ebdf009e01f